dimagi_lake/aggregation/aggregation_helpers/agg_flwc_administration.py

```diff
@@ -0,0 +1,138 @@
+from datetime import datetime
+
+from dimagi_lake.aggregation.aggregation_helpers.base_helper import \
+    BaseAggregationHelper
+from dimagi_lake.utils import clean_name
+from spark_session_handler import SPARK
+
+
+class AggFlwcAdministartionHelper(BaseAggregationHelper):
+
+
+    @property
+    def source_tablename(self):
+        return "flwc_management_flwc_administration"
+
+    def aggregate(self):
+        df = self.pre_process()
+        df.show()
+        df = self.apply_transformation(df)
+
+        return self.enforce_schema(df)
+
+    @property
+    def columns(self):
+        return {
+            "infrastructure_available": (
+                'form',
+                'flwc_information_and_infrastructure_group',
+                'infrastructure_at_flwc_details_group',
+                'infrastructure_available'
+            ),
+            "gme_available": (
+                "form",
+                "essentials_at_flwc_details_group",
+                "growth_measuring_equipments_details_group",
+                "gme_available"
+            ),
+            "utility_items_list": (
+                "form",
+                "essentials_at_flwc_details_group",
+                "other_utility_items_details_group",
+                "utility_items_list"
+            ),
+            "timeend": (
+                "form",
+                "meta",
+                "timeEnd"
+            )
+        }
+
+    def pre_process(self):
+        df = SPARK.sql(f"""
+        SELECT
+            flwc_id,
+            supervisor_id,
+            project_id,
+            district_id,
+            state_id,
+            {self.get_column('timeend')} as timeend,
+            int(array_contains(split({self.get_column('infrastructure_available')}, ' '), 'safe_drinking_water')) as safe_drinking_water,
```

Comment:
Because of the flattening of the data frame. Column names become longer. Therefore, accessing column like this `self.get_column('infrastructure_available')`

Reply:
this is mainly a readability thing?

Reply:
yes